morf-core/src/main/java/org/alfasoftware/morf/sql/SelectStatement.java

```diff
@@ -477,14 +477,31 @@ public SelectStatement allowParallelDml() {
    * @param customHint representation of a custom hint
    *
    * @return this, for method chaining.
+   * @deprecated See {@link org.alfasoftware.morf.sql.CustomHint}

```

Comment:
This should link to the new `withDialectSpecificHint` method, so that developers can quickly find a replacement.

Reply:
Done. Waiting to fix the other comments before pushing all the changes.

Reply:
Done